sdk/translation/Azure.AI.Translation.Text/samples/Sample2_Translate.md

```diff
@@ -0,0 +1,366 @@
+# Translate
+
+## Create a `TextTranslationClient`
+
+To create a new `TextTranslationClient`, you will need the service endpoint and credentials of your Translator resource. In this sample, however, you will use an `AzureKeyCredential` and region, which you can create with an API key.
+
+```C#
+AzureKeyCredential credential = new("<apiKey>");
+TextTranslationClient client = new(credential, "<region>");
+```
+
+The values of the `apiKey` and `region` variables can be retrieved from environment variables, configuration settings, or any other secure approach that works for your application.
+
+### Translate text
+Translate text from known source language to target language.
+```C#
+try
+{
+    string from = "en";
+    string targetLanguage = "cs";
+    string inputText = "This is a test.";
+
+    Response<IReadOnlyList<TranslatedTextItem>> response = await client.TranslateAsync(targetLanguage, inputText, from: from).ConfigureAwait(false);
+    IReadOnlyList<TranslatedTextItem> translations = response.Value;
+    TranslatedTextElement translation = translations.FirstOrDefault();
+
+    Console.WriteLine($"Detected languages of the input text: {translation?.DetectedLanguage?.Language} with score: {translation?.DetectedLanguage?.Score}.");
+    Console.WriteLine($"Text was translated to: '{translation?.Translations?.FirstOrDefault().To}' and the result is: '{translation?.Translations?.FirstOrDefault()?.Text}'.");
+}
+catch (RequestFailedException exception)
+{
+    Console.WriteLine($"Error Code: {exception.ErrorCode}");
+    Console.WriteLine($"Message: {exception.Message}");
+}
+```
+
+### Translate with auto-detection
+You can ommit source languge of the input text. In this case, API will try to auto-detect the language.
```

Comment:
I feel like this should be the first sample. at the end of the section we should explain limitations of auto-detection, then the following sample should show how to explicitly provide largauge.

Reply:
I went from the simplest use case - you know "source" and "target" language and the text. I would say, that is the most common case. But I can re-arrange the examples as needed.